image_builder/progress.py

```diff
@@ -42,9 +42,9 @@ def __init__(self, name):
         self.status = PendingStatus()
 
     def __str__(self):
-        output = f"*{self.name}*: {self.status}"
+        output = f"*{self.name.capitalize()}*: {self.status}"
```

Comment:
Maybe we can capitalise the status as well?
```suggestion
        output = f"*{self.name.capitalize()}*: {self.status.capitalize()}"
```